weave-js/src/components/Checkbox/Checkbox.tsx

```diff
@@ -46,6 +46,7 @@ export const Checkbox = ({
           'rounded border-[1.5px]  dark:border-moon-250',
           'focus-visible:outline focus-visible:outline-[2px] focus-visible:outline-teal-500',
           checked ? 'border-moon-800' : 'border-moon-500',
+          props.disabled && 'cursor-not-allowed opacity-30',
```

Comment:
No longer used in this pr, but looks like: 
<img width="192" alt="Screenshot 2024-07-10 at 4 21 41 PM" src="https://github.com/wandb/weave/assets/19414170/1adebaf2-7930-4d46-ab0c-c1f792fa1d01">